packaging/make_package

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/bash
+#
+#we want to name the binary grafana-agent not agent-linux-amd64
+#
+fpm.ruby2.6 -s dir -t rpm -f \
+  --package ../dist \
+  --log warn \
+  -v 0.4.0 \
+  -n grafana-agent \
+  --license "Apache 2.0" \
+  --vendor "Grafana Labs" \
+  --url "https://github.com/grafana/agent" \
+  --after-install after_install.sh \
+  --before-remove before_remove.sh \
+  --config-files etc/grafana-agent.yaml \
+  --config-files etc/sysconfig/grafana-agent  \
+  ../dist/agent-linux-amd64=/usr/bin/grafana-agent \
+  var=/ grafana-agent.service=/usr/lib/systemd/system/grafana-agent.service
```

Comment:
```suggestion
  grafana-agent.service=/usr/lib/systemd/system/grafana-agent.service
```
I think var=/ needs to be removed as there is no var directory when I tested locally.

Reply:
In fact, I think the var is missing from the repository. When I try and test the package with the centos-systemd image in [] I see an error relating to the `var` directory:

```
$ # From the packaging directory
$ docker run -it --tmpfs /run --tmpfs /run/lock -v /sys/fs/cgroup:/sys/fs/cgroup:ro -v "$(git rev-parse --show-toplevel)"/dist:/dist quay.io/cortexproject/centos-systemd:latest rpm -i /dist/grafana-agent-0.4.0-1.x86_64.rpm

chown: cannot access '/var/lib/grafana-agent': No such file or directory
warning: %post(grafana-agent-0.4.0-1.x86_64) scriptlet failed, exit status 1
```